docs/source/troubleshooting.rst

```diff
@@ -91,3 +91,9 @@ An example YAML specification looks like this:
 When running DeepRC I get TypeError: can't concat str to bytes
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 This error occurs when h5py version 3 or higher is used. Try using version 2.10.0 or lower.
+
+I am trying to run a report, but it gives no results and no errors. What happened?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+To ensure that large analyses do not crash if one of the reports failed (for example, due to
+incorrectly specified parameters), reports are run in a 'safe mode' and errors are converted to warnings.
```

Comment:
"errors are converted to warnings" -> "reports are run in a 'safe mode' so that errors do not stop the execution." or sth like that.. also it will not crash because of incorrectly specified parameters (checked at parsing time?!), but maybe if the results cannot be computed for some reason...

Reply:
right, I updated this. I actually wrote this Q+A several weeks ago but didnt bother making a separate branch for it, I remember I formulated it like this because the report I tried to run failed due to incorrectly specified params, but it was a parameter that couldn't be checked during parsing because it was dataset-dependent (I think it was the FeatureComparison report), but this formulation is better for most cases indeed